concepts/classes/about.md

```diff
@@ -0,0 +1,231 @@
+# classes
+
+Classes are a central concept in [object-oriented programming][object-oriented-programming].
+[In Crystal, everything is an object][everything-is-an-object], every object has a type, and it can respond to some methods.
+Classes are blueprints to create objects, providing initial values for state, like variables, and implementations of behavior, like methods.
+When creating an object from a class, the object is an instance of that class, the instance holds all the instance variables and states.
+
+## Class definition
+
+A class is defined using the `class` keyword, followed by the name of the class written in `UpperCamelCase`.
+
+```crystal
+class Account
+end
+```
+
+## Creating an instance
+
+To [create an instance of a class][new-initialize], you call the class name with the `new` method.
+
+```crystal
+class Account
+end
+
+account = Account.new
+```
+
+In the example above the variable `account` is an instance of the class `Account`.
+
+When creating an instance, there is a special method called `initialize` that is a constructor.
+It is called when an instance is created.
+The constructor allows you to set the initial state of the instance.
+The `initialize` method can take arguments, which are passed when creating an instance.
+
+```crystal
+class Account
+  def initialize(number)
+    p 5 + number
+  end
+end
+
+Account.new(4)
+# => 9
+```
+
+````exercism/note
+The `initialize` method cant return a value, instead, when calling new the instance is returned.
+
+```crystal
+class Account
+  def initialize(number)
+    5 + number
+  end
+end
+
+Account.new(4)
+# => #<Account:0x7f5dc33dcea0>
+````
+
+## Instance methods
+
+An instance method is a method that is available to an instance of a class.
+And can be called on an instance of a class.
+Instance methods are defined using the `def` keyword, followed by the name of the method and they need to be defined inside the class definition.
+
+```crystal
+class Account
+  def balance
+    100
+  end
+end
+```
+
+## Instance variables
+
+[Instance variables][instance-variable] are variables that are available in all methods of the instance.
+They can be initialized in the `initialize` method.
+They are defined using the `@` prefix.
```

Comment:
It might be worth noting that you typically can't access instance variables by default, you must define a method which exposes the instance variable.